scss/styles.scss

```diff
@@ -77,17 +78,64 @@
 }
 
 .image-container {
+  max-width: 261px;
   position: relative;
+  width: 100%;
 
   &::before {
     bottom: 55px;
```

Comment:
maybe it would be better to use top, if the image changes its height the badge will always be in the same place